memstore/backfill_test.go

```diff
@@ -348,16 +382,19 @@ var _ = ginkgo.Describe("backfill", func() {
 
 		// column should not be forked again.
 		Ω(forkedColumn).Should(Equal(backfillCtx.new.Columns[4]))
-	})
 
+		logger.Infof("Test writePatchValueForUnsortColumn should work Finished")
+	})
+/*
 	ginkgo.It("apply backfill patch should work", func() {
```

Comment:
are thee tests commented out...?

Reply:
was in testing phase, the latest version should be restored